Src/BootCamp.Chapter/Checks.cs

```diff
@@ -17,25 +17,26 @@ public static class Checks
         public static int PromptInt(string message)
         {
             // To do: call your implementation. 
-            return 0;
+            return (int)Program.GetFloatInput(message);
```

Comment:
This works, but you had non-functional requirements to do it in a separate function and not via casting :)